front-end/app/checkout.component/payment-processing.component.ts

```diff
@@ -43,6 +43,7 @@ export class PaymentProcess implements OnInit, OnDestroy {
                                     email: buyerDetails.email,
                                     phone: buyerDetails.phone
                                 },
+                                note: buyerDetails.note,
```

Comment:
The note is part of the order, not the buyer details. The buyer details will be persisted between orders, but the notes should not be. Maybe notes should be added on the basket screen considering they are to do with the basket?